molo/core/tests/test_models.py

```diff
@@ -345,6 +345,17 @@ def test_commenting_enabled(self):
             commenting_state=constants.COMMENTING_DISABLED)
         self.assertFalse(article_3.is_commenting_enabled())
 
+    def test_commenting_anonymous_alias(self):
```

Comment:
can we also test that when making a comment, that the anonymous alias is used, and that it is the correct one being used. Can we also do a test for a translated version of anonymous?

Reply:
yes I will addd that